packages/new-deploy/src/DeployContext.ts

```diff
@@ -113,26 +113,92 @@ export default class DeployContext extends MultiProvider<config.Domain> {
     this._data = config.addBridge(this.data, name, bridge);
   }
 
-  pushVerification(
```

Comment:
this is added back at the bottom, just moving things around in the file don't mind me 😬

Reply:
reordering makes reviewing changes near-impossible. move it back and reorder in a separate PR